README.md

```diff
@@ -36,6 +36,21 @@ mapB.sync(mapC);
 mapC.sync(mapA);
 mapC.sync(mapB);
 ```
+## Offset
+You can synchronize not only the centers, but other points, using the option `offsetFn`.
+The parameters send to the function are `(center, zoom, referenceMap, targetMap)`, and it must return the equivalent center to produce your offset. That means, the center to pass to setView.
+
+In most cases, you can use the factory `offsetHelper`, that accepts two arrays of two elements each `(ratioRef, ratioTgt)`. The meaning of this array is the relative position -relative to the top left corner and the whole size- in the map container of the point to synchronize. (1 is for the whole width or height). Values greater than 1 or less than 0 work fine.
+
```

Comment:
From this paragraph, I wonder:
 - can you add/is it beneficial to have a use case + example of a sync scenario which is not possible by using the `offsetHelper`.
- "Values greater than 1 or less than 0 work fine." -> might be  good to explain this a little more/give an example with the two points + what they mean.